### DISCUSSION TITLE
termux下alist修改admin账户的名称后登录显示找不到用户，重新生成密码也无用

### DISCUSSION THREAD

**Discussion by waitfortea** [AlistGo/alist#6627] [Q&A]
![image](https://github.com/alist-org/alist/assets/78167600/0dbb21b8-11b5-402d-ae45-5ed206dd1d38)
![9f4519da510586a52c9a1a3b24a6c5d](https://github.com/alist-org/alist/assets/78167600/f07c156d-6122-401d-b8ed-76712ef09cf8)
![image](https://github.com/alist-org/alist/assets/78167600/7dee7fea-26e8-425c-914c-be05e6fadaeb)
注意这个时间在我重置密码之后，还是找不到用户
alist版本 3.35

**Comment by pongfcnkl** [comment]
用户名前面有个空格
